runtime/mandala/src/lib.rs

```diff
@@ -2588,8 +2591,8 @@ mod tests {
 	#[test]
 	fn check_call_size() {
 		assert!(
-			core::mem::size_of::<Call>() <= 230,
-			"size of Call is more than 230 bytes: some calls have too big arguments, use Box to \
+			core::mem::size_of::<Call>() <= 280,
+			"size of Call is more than 280 bytes: some calls have too big arguments, use Box to \
 			reduce the size of Call.
 			If the limit is too strong, maybe consider increasing the limit",
```

Comment:
You can Box the argument in a call that causes to the `Call` enum to be too large... In this case the limit is only slightly changed so I don't feel strongly but see https://github.com/open-web3-stack/open-runtime-module-library/pull/598

Reply:
thanks. gonna keep the current approach.